docker/R/lib/api_get_status.rb

```diff
@@ -58,16 +58,20 @@
 
 result = {}
 result[:status] = false
+get_count = 0
 begin
+  get_count += 1
   a = RestClient.get "#{options[:host]}/analyses/#{options[:analysis_id]}/status.json" , {accept: :json}
-  # TODO: retries?
   raise 'Could not create datapoint' unless a.code == 200
-
+  puts "#{__FILE__} success! get_count: #{get_count}"
   a = JSON.parse(a, symbolize_names: true)
   result[:status] = true
   result[:result] = a[:analysis][:run_flag]
 rescue => e
+  sleep(2)
```

Comment:
I think this should be at least 10 seconds... that way it has a full 100 seconds to try and recovery.